Code/Framework/AzCore/AzCore/Math/Quaternion.inl

```diff
@@ -66,6 +66,20 @@ namespace AZ
         return Quaternion(Simd::Vec4::FromVec3(v.GetSimdValue()));
     }
 
+    AZ_MATH_INLINE Quaternion Quaternion::CreateFromEulerDegreesXYZ(const Vector3& eulerRadians)
+    {
+        return CreateFromEulerRadiansXYZ(Vector3DegToRad(eulerRadians));
+    }
+
+    AZ_MATH_INLINE Quaternion Quaternion::CreateFromEulerDegreesYXZ(const Vector3& eulerRadians)
+    {
+        return CreateFromEulerRadiansYXZ(Vector3DegToRad(eulerRadians));
+    }
+
+    AZ_MATH_INLINE Quaternion Quaternion::CreateFromEulerDegreesZYX(const Vector3& eulerRadians)
+    {
+        return CreateFromEulerRadiansZYX(Vector3DegToRad(eulerRadians));
+    }
```

Comment:
update: Parameter names need to be updated to `eulerDegrees`

note: We should also maybe mention something about the range the degrees should be in (usually we might expect +/-180 degrees? Will things break if we pass a value outside that range or is it handled gracefully? Could be a useful test to add to verify.